milli/Cargo.toml

```diff
@@ -54,7 +54,10 @@ big_s = "1.0.2"
 insta = "1.21.0"
 maplit = "1.0.2"
 md5 = "0.7.0"
-rand = "0.8.5"
+rand = {version = "0.8.5", features = ["small_rng"] }
```

Comment:
Why have we enabled this feature?

Reply:
In tests I want a random number generator that I can give a seed to, and I *think* I had to use `small_rng` for that.